cmd/update-operator/main.go

```diff
@@ -25,12 +27,16 @@ var (
 )
 
 func main() {
+	flag.Var(&preRebootActions, "require-pre-reboot", "list of comma-separated kubernetes annotations required to be set to 'true' before the operator will authorize a reboot")
```

Comment:
Drop mention of authorization.

before-reboot-annotations - "List of comma separated Kubernetes node annotations that must be set to true before a reboot is allowed."